packages/meditrak-app/ios/TupaiaMediTrak/TupaiaMediTrak.entitlements

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+<dict>
+	<key>com.apple.developer.associated-domains</key>
+	<array>
+		<string>applinks:datatrak.tupaia.org?mode=developer</string>
+		<string>webcredentials:tupaia.org</string>
+		<string>webcredentials:datatrak.tupaia.org</string>
+		<string>applinks:waitp-1500-open-ios-links-in-app-datatrak.tupaia.org?mode=developer</string>
+		<string>applinks:localhost:5173?mode=developer</string>
```

Comment:
For production, this will only include:

- applinks:datatrak.tupaia.org
- webcredentials:datatrak.tupaia.org
- webcredentials:tupaia.org (this one’s arguably optional, but means one less step to remember when we eventually implement single sign-on)